.gitignore

```diff
@@ -142,3 +142,5 @@ ingestion_server/ingestion_server/lock
 .vscode
 
 openapi.yaml
+Pipfile
+Pipfile.lock
```

Comment:
`Pipfile` and `Pipfile.lock` must not be ignored.

Reply:
For more context, these two files are used for defining and locking our dependencies and are required for our project!

Reply:
Committed the changes.